docs/DeployConfigurations.rst

```diff
@@ -152,31 +217,20 @@ In this scenario, Nextcloud is installed within a container, and a separate Daem
 			Daemon --> Containers
 
 			state Containers {
-				[*] --> Nextcloud : /var/run/docker.sock
-
-				state Nextcloud {
-					Daemon2 --> Containers2
-
-					state Containers2 {
-						ExApp1
-						--
-						ExApp2
-						--
-						ExApp3
-					}
-				}
+				[*] --> DockerSocketProxy : /var/run/docker.sock
+				Nextcloud --> DockerSocketProxy: by port
+				--
+				DockerSocketProxy --> ExApp1
+				DockerSocketProxy --> ExApp2
```

Comment:
cannot judge the changes above, (no preview available)

Reply:
![Screenshot_6](https://github.com/cloud-py-api/app_api/assets/29692256/b56bdd9f-df55-44e7-b770-1882fb4e6708)